student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Student < Person
+  def initialize(_classrom, age, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
+    @classroom = classroom
+  end
```

Comment:
- [ ] you're creating the instance vairable `@classroom` using `classroom` however the name of the argument you're passing is `_classrom` please make sure fix this.

Reply:
Thank you for your review, I'm extremely sorry for my mistake, I'll fix it ASAP